terraform/production/outputs.tf

```diff
@@ -10,6 +10,10 @@ output "public_subnet_ids" {
   value = module.networking.public_subnets[*].id
 }
 
+output "cpm_private_subnet_ids" {
```

Comment:
Do we need these outputs?

Reply:
This is so the CPM repo can access these.